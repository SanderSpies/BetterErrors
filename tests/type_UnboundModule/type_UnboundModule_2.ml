module HelloWorld = struct
end

open HulloWorld
