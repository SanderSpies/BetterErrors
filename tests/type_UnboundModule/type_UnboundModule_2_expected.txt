[36mtests/type_UnboundModule/type_UnboundModule_2.ml:4 5-15
[0m1 | module HelloWorld = struct
2 | end
3 | 
4 | open [31;4mHulloWorld[0m

[31mError[0m: Module `HulloWorld` not found in included libraries.
Hint: did you mean `HelloWorld`?
